commit 4dd893d832cf4da5409e1fecea9c4f6452a93f2b
Author: Eric Biggers <ebiggers@google.com>
Date:   Mon Jul 22 09:26:23 2019 -0700

    fs-verity: implement FS_IOC_MEASURE_VERITY ioctl
    
    Add a function for filesystems to call to implement the
    FS_IOC_MEASURE_VERITY ioctl.  This ioctl retrieves the file measurement
    that fs-verity calculated for the given file and is enforcing for reads;
    i.e., reads that don't match this hash will fail.  This ioctl can be
    used for authentication or logging of file measurements in userspace.
    
    See the "FS_IOC_MEASURE_VERITY" section of
    Documentation/filesystems/fsverity.rst for the documentation.
    
    Reviewed-by: Theodore Ts'o <tytso@mit.edu>
    Reviewed-by: Jaegeuk Kim <jaegeuk@kernel.org>
    Signed-off-by: Eric Biggers <ebiggers@google.com>

diff --git a/fs/verity/Makefile b/fs/verity/Makefile
index 04b37475fd28..6f7675ae0a31 100644
--- a/fs/verity/Makefile
+++ b/fs/verity/Makefile
@@ -3,5 +3,6 @@
 obj-$(CONFIG_FS_VERITY) += enable.o \
 			   hash_algs.o \
 			   init.o \
+			   measure.o \
 			   open.o \
 			   verify.o